src/Integration/Upcasting/UpcastingEventsTest.php

```diff
@@ -38,7 +38,7 @@ public function upcasting_works()
         $serializer = new UpcastingMessageSerializer(new ConstructingMessageSerializer(), $upcaster);
 
         $message = iterator_to_array($serializer->unserializePayload($payload))[0];
-        $expected = $defaultDecorator->decorate(new Message(new UpcastedEventStub('upcasted')))->withHeader('version', 1);
+        $expected = $defaultDecorator->decorate(new Message(new UpcastedPayloadStub('upcasted')))->withHeader('version', 1);
```

Comment:
Move these calls to different lines so it doesn't goes over the 120 soft limit. This reads better in code reviews.

```suggestion
        $expected = $defaultDecorator
                ->decorate(new Message(new UpcastedPayloadStub('upcasted')))
                ->withHeader('version', 1);
```